File: Triatlon-CS/Networking/ResultDTO.cs
using System;

namespace Networking
{
    [Serializable]
    public class ResultDTO
    {
        public ResultDTO(int id, int idParticipant, int idStage, double score)
        {
            Id = id;
            IdParticipant = idParticipant;
            IdStage = idStage;
            Score = score;
        }

        public int Id { get; set; }

        public int IdParticipant { get; set; }

        public int IdStage { get; set; }

        public double Score { get; set; }

        public override string ToString()
        {
            return string.Format("Id = {0} | participant = {1} | stage = {2} | score = {3}", Id, IdParticipant, IdStage, Score);
        }
    }
}

File: Triatlon-Java/ChatNetworking/src/main/java/triathlon/network/dto/ResultDTO.java
package triathlon.network.dto;

import triathlon.model.Entity;

import java.io.Serializable;

public class ResultDTO extends Entity<Integer> implements Serializable {
    private Integer participantId;
    private Integer stageId;
    private Double score;

    public ResultDTO(Integer participantId, Integer stageId, Double score) {
        this.participantId = participantId;
        this.stageId = stageId;
        this.score = score;
    }

    public Integer getParticipantId() {
        return participantId;
    }

    public Integer getStageId() {
        return stageId;
    }

    public Double getScore() {
        return score;
    }

    @Override
    public String toString() {
        return "ResultDTO[ participant: " + participantId + " stage: " + stageId + " score: " + score + "]";
    }
}
